app/common/util/pom.xml

```diff
@@ -117,4 +122,16 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-Xmx200m</argLine>
```

Comment:
I think this removes the code coverage setup by jacoco, you can change the `basepom.test.memory` property instead